packaging/osx/hostfxr/resources/cs.lproj/conclusion.html

```diff
@@ -1,22 +1,14 @@
-
+﻿
 <!DOCTYPE html>
 <html lang="en">
 <head>
     <meta charset="utf-8" />
-    <title styles="color:white">Congratulations! You've successfully installed .NET Core CLI!</title>
+    <title styles="color:white">Congratulations! You've successfully installed .NET Core HostFxr!</title>
 </head>
 <body>
         <div align="center" style="font-family: Helvetica;">  
             <h1>The installation was successful.</h1>
-            <p>.NET Core CLI was successfully installed.</p>
-        <h2>
-            Install dependencies
-        </h2>
-        <p>
-        In order to be able to use .NET Core on OS X, you need to install <b>OpenSSL</b> version <b>1.0.1/1.0.2</b>. 
-        There are many ways to install/update your libssl. Using <a href="http://www.brew.sh">Homebrew</a> is the easiest. 
-        You can view the instructions <a href="http://brewformulas.org/Openssl">here</a> or if you're updating, <a href="https://github.com/dotnet/coreclr/blob/63766f74c4a641a274cd2933b9b7fd7bbddef2dd/Documentation/building/osx-instructions.md#openssl">on this page</a>.
-        </p>
+            <p>.NET Core Runtime was successfully installed.</p>
```

Comment:
I further assume that this change was intentional (though unrelated to the description)